docs/developers/vcpkg.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Setup depends via vcpkg
+nav_order: 9
+parent: Developer Overview
+---
+
+# Setup static native depends via vcpkg
+
+## Setup build depends
+
+Please install build depends to compile all libraries:
+
+``` sh
+sudo $GLUTEN_REPO/dev/vcpkg/setup-build-depends.sh
+```
+
+You can install the following tools manually on unsupport distro.
+
+* zip
+* tar
+* wget
+* curl
+* git >= 2.7.4
+* gcc >= 9
+* pkg-config
+* autotools
+* flex >= 2.6.0
+* bison
+
```

Comment:
in my local tests, it requires another step to enable the new gcc via: `source /opt/rh/devtoolset-9/enable`
otherwise the script will fail and exit due to no gcc found